drake/doc/no_push_to_origin.rst

```diff
@@ -0,0 +1,39 @@
+*****************************************
+Avoid Accidental Pushes to the Repository
+*****************************************
+
+The Issue
+=========
+
+We want to prevent developers from accidentally pushing changes into RobotLocomotion/drake. This is accomplished by:
+
+   1. Everybody agreeing in a naming convention for the respositories.
```

Comment:
i don't think we need this.  everyone's workflow is different (mine is pretty complicated).